test/integration/labelled-namespace-not-enabled/00-assert.yaml

```diff
@@ -1,3 +1,12 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 30
+collectors:
+  - type: pod
+    selector: app=test
```

Comment:
hey @YashPimple this is correct, however, the ticket also requires to collection of the logs of the lifecycle controller pod, to do so you can use the selector `control-plane: lifecycle-operator` and do the same as line 7 and 8